packages/main/test/specs/MultiComboBox.spec.js

```diff
@@ -785,6 +785,96 @@ describe("MultiComboBox general interaction", () => {
+
+			const mcb = await browser.$("#mcb");
+			const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#mcb");
+			const popover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+
+			await mcb.click();
+			await mcb.keys("F4");
+
+			const listItem = await popover.$("ui5-list").$("ui5-li");
+
+			assert.equal(await listItem.getProperty("focused"), true, "The first item is focused");
+
+			await mcb.keys("ArrowDown");
+			await mcb.keys("Space");
+			await mcb.keys("F4");
+			await mcb.keys("F4");
+
+			const listItem2 = await popover.$("ui5-list").$$("ui5-li")[1];
+
+			assert.equal(await listItem2.getProperty("focused"), true, "The second item is focused as it is selected");
+		});
+
+		it ("Alt + Down should focus the corresponding item to the token from which the combination is pressed", async () => {
+			await browser.url(`http://localhost:${PORT}/test-resources/pages/MultiComboBox.html`);
+
+			const mcb = await browser.$("#mcb-items");
+			const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#mcb-items");
+			const popover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+			let tokens = await mcb.shadow$$(".ui5-multi-combobox-token");
+
+			await tokens[2].click();
+			await tokens[2].keys(["Alt", "ArrowDown"]);
+
+			let listItem = await popover.$("ui5-list").$$("ui5-li")[3];
+
+			assert.equal(await listItem.getProperty("focused"), true, "The selected item corresponding to the token is focused");
+		});
+
+		it ("Alt + Down should focus the first item if no selected filtered items are present", async () => {
+			await browser.url(`http://localhost:${PORT}/test-resources/pages/MultiComboBox.html`);
+
+			const mcb = await browser.$("#multi-acv");
+			const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#multi-acv");
+			const popover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+
+			await mcb.click();
+			await mcb.keys(["Alt", "ArrowDown"]);
+
+			let listItem = await popover.$("ui5-list").$$("ui5-li")[0];
+
+			assert.equal(await listItem.getProperty("focused"), true, "The first item is focused");
+		});
+
+		it ("Alt + Down should not filter items", async () => {
+			await browser.url(`http://localhost:${PORT}/test-resources/pages/MultiComboBox.html`);
+
+			const mcb = await browser.$("#mcb");
+			const input = await mcb.shadow$("input");
+			const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#mcb");
+			const popover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+
+			await input.click();
+			await input.keys(["a", "b"]);
+			await input.keys(["Alt", "ArrowDown"]);
+
+			let listItem = await popover.$("ui5-list").$$("ui5-li")[0];
+
+			assert.equal(await listItem.getProperty("focused"), true, "The items are not filtered and the first item is focused");
```

Comment:
Here is the same, they are always not filtered. In the test description above as well